contracts/Crowdsale/BasicCrowdsale.sol

```diff
@@ -19,29 +19,17 @@ contract BasicCrowdsale is MintedCrowdsale, FinalizableCrowdsale {
     uint256 constant MAX_CONTRIBUTION_AMOUNT = 250 ether;
 
     uint256 public constant PRIVATE_SALE_CAP = 26000000 * (10 ** 18);
-    uint256 public constant PRIVATE_SALE_DURATION = 60 days;
-
-    uint256 public constant PRESALE_CAP = PRIVATE_SALE_CAP + (5000000 * 10 ** 18);
-    uint256 public constant PRESALE_BONUS = 30; // % bonus from default rate
-    uint256 public constant PRESALE_DURATION = 14 days;
- 
-    uint256 public constant PHASE_1_CAP = PRESALE_CAP + (10000000 * 10 ** 18);
-    uint256 public constant PHASE_1_BONUS = 20; // % bonus from default rate
-    uint256 public constant PHASE_1_DURATION = PRESALE_DURATION + 14 days;
-
-    uint256 public constant PHASE_2_CAP = PHASE_1_CAP + (12500000 * 10 ** 18);
-    uint256 public constant PHASE_2_BONUS = 10; // % bonus from default rate
-    uint256 public constant PHASE_2_DURATION = PHASE_1_DURATION + 14 days;
- 
-    uint256 public constant PHASE_3_CAP = PHASE_2_CAP + (12500000 * 10 ** 18);
-    uint256 public constant PHASE_3_BONUS = 5; // % bonus from default rate
-    uint256 public constant PHASE_3_DURATION = PHASE_2_DURATION + 14 days;
+    uint256 public constant PRIVATE_SALE_DURATION = 30 days; // to be calculated according to deployment day; the end date should be 30 May
+
+    uint256 public constant MAIN_SALE_DURATION = 60 days;
+    uint256 public mainSaleDurantionExtentionLimit = 120; //max days the duration of the ICO can be extended
```

Comment:
Initial limit needs to be 60days. 60days main sale duration + 60 days max extension = 120 max total duration